src/xo-app/pool/tab-patches.js

```diff
@@ -1,13 +1,160 @@
-import _ from 'intl'
+import ActionButton from 'action-button'
+import Component from 'base-component'
 import React from 'react'
+import SortedTable from 'sorted-table'
+import TabButton from 'tab-button'
+import _ from 'intl'
+import forEach from 'lodash/forEach'
+import isEmpty from 'lodash/isEmpty'
+import map from 'lodash/map'
```

Comment:
Workaround in ```.emacs```:
```lisp
(custom-set-variables
 '(sort-fold-case t t))
```
Because emacs is more configurable than any text editor and there are updates unlike debian. :heart: